imod/mf6/out/common.py

```diff
@@ -19,3 +19,13 @@ def _to_nan(a: FloatArray, dry_nan: bool) -> FloatArray:
     if dry_nan:
         a[a == -1e30] = np.nan
     return a
+
+
+def get_header_advanced_package(
+    headers: Dict[str, List[Any]],
+) -> Any:
+    for key, header in headers.items():
+        # multimodels have a gwf-gwf budget for flow-ja-face between domains
+        if not "flow-ja-face" in key and "gwf_" in key:
+            return header[0]
+    return None
```

Comment:
I don't really get what this function is doing:

Why does it return ``header[0]`` and not ``header``? Does this mean the function name is incorrect, or are we actually returning the first header in the list? If the latter:

```suggestion
def get_header_advanced_package(
    headers: Dict[str, List[Any]],
) -> Any:
    for key, header_ls in headers.items():
        # multimodels have a gwf-gwf budget for flow-ja-face between domains
        if not "flow-ja-face" in key and "gwf_" in key:
            return header_ls[0]
    return None
```

In that case, maybe better function name is: ``get_first_header_advanced_package``

UPDATE: I see in other part of the codebase:
``for key, header_list in headers.items():``
So I think that is clearer here.